inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -57,6 +57,14 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
 
     virtual void Serialize();
 
+    static void Compare(const std::vector<std::pair<ngraph::element::Type, std::vector<std::uint8_t>>> &expected,
```

Comment:
I think it will be better to use old function, just add default parameters 
```
static void Compare(const std::vector<std::pair<ngraph::element::Type, std::vector<std::uint8_t>>> &expected,
                        const std::vector<InferenceEngine::Blob::Ptr> &actual,
                        float relThreshold, float absTreshold = -1.0f);

```

In this case we can remove new ones

Reply:
@iefode Could you please elaborate more about your point? Sorry, I couldn't understand exactly. This function definition is for new absolute threshold. 
You mean that your suggestion can remove the function body in layer_test_utils.cpp?